### DISCUSSION TITLE
Recommended Capture card setup for 4K TV

### DISCUSSION THREAD

**Discussion by stickmanbrad** [awawa-dev/HyperHDR#612] [General]
Hiya,

Really hoping you can help. I currently am using a Zasluke 4K capture card with hdcp 2.2 / EDID. It’s been good initially, but I’ve noticed the colours aren’t very accurate, and there can be delays between what’s on the TV and the led strips. I’m running the image back to the TV through the pass through built into the capture card.

My question is, can a new capture card be recommended that still works with all the content (so the hdcp 2.2 and edip), more colour accurate and support my 4K TV? I’d be really grateful if something could be suggested that I can get delivered to the Uk.

kind regards in advanced

**Comment by awawa-dev** [comment]
Hi
If HDCP 2.2 is a strict requirement then I don't know of any modern and good USB 3.0 grabber with a loop (all in one) that I can recommend. Newer models handle HDCP 2.2 in a peculiar way: they can pass video protected by HDCP 2.2, but they can't capture it, probably because of some "dark" agreements they had to accept when accessing the chips and their API. At least officially. Unofficially, some grabbers have an undocumented feature that allows HDCP capture. More in the comments section of this thread (at the end) https://www.hyperhdr.eu/2021/01/grabbers-for-hyperhdr-part-2-ezcap-320.html If you are OK with using HDMI splitter then check this thread (there are a lot of HDMI splitters tested by users, not only ms2130): https://github.com/awawa-dev/HyperHDR/discussions/499

**Reply by stickmanbrad** [reply]
Can I check.. just because I did just order one. Will the MS2130 work with content on hdcp 2.2 if I run through a splitter like the ezcoo one?

kind regards

**Reply by stickmanbrad** [reply]
I’m now worried that maybe I should have ordered the Excap 320 instead lol.. let me know your thoughts. I know the Zasluke one just isn’t really giving me the colour accuracy, so I figured I’d just take the recommendation for a new one if there is one.

Thanks again in advanced

**Reply by awawa-dev** [reply]
> Will the MS2130 work with content on hdcp 2.2 if I run through a splitter like the ezcoo one?

Yes, in the mentioned thread, users are using such a configuration. A good HDMI splitter will remove HDCP on the output to the grabber. It seems to me that the Ezcap 320 is a discontinued product, and its colors in HDR are not the best anyway: https://www.hyperhdr.eu/2022/04/usb-grabbers-hdr-to-sdr-quality-test.html#chapter36

**Reply by stickmanbrad** [reply]
I was going to order the EZCOO [EZ/EU-SP12H28S] unless you had a better recommendation. It’s such a pain taking the tv off the wall at this high so I basically want the best solution to not have to think about it for another couple of years 😂. So I’ve ordered the MS2130 (which was very cheap so I’m scepitcal lol). Just need advice on the best splitter for the hdcp 2.2, and might as well future proof for 4k 120hz on the tv output. 

Kind regards for any advice.

**Reply by stickmanbrad** [reply]
Just looking for the best combination. You appear to have been quite happy with the MS2130 based on your recent blog post. Assuming that is the best capture card at this point, just hoping for some advice on the splitter. I’ve also gone ahead and ordered some MH-ET D1 Mini’s so I avoid any issues with that handshake again 😂.

Really grateful for any further advice

**Reply by awawa-dev** [reply]
Like I said, if you can't disable HDCP, then you're stuck with a HDMI splitter anyway. In the case of ms2130, the splitter will perform additional tasks such as downscale with HDMI2.1 materials and force HDR, which normally this grabber does not support. Yes it's cheap but it's a grabber that officially supports only HDMI1.4 and a digital signal, so the integrated components are cheap, it also has built-in efficient cooling, although, as the experience with the older ms2109 teaches, you have to be careful here because there may be a cost cut here in the future. With no HDCP requirement, I can recommend the Elgato HD60X which I use in my home setup. You have all the information in the attached threads, but you have to make your own decision.

**Reply by stickmanbrad** [reply]
Any advice on how the MS2130 will compare to my current Zasluke capture card? At the moment as I said it’s running over pass through. I intended getting the splitter anyways. But just interested as to whether I’ll see better latency and better quality of colour accuracy over my current setup?

**Reply by awawa-dev** [reply]
You can perform automatic calibration for Zasluke in HyperHDR and compare the results available in the logs right after it, e.g. with those I put in the first post in the thread about ms2130 (in the details section)

**Comment by Vizavi** [comment]
Hey, i've order an EVGA Precision X1 pro, as for specs - it's fantastic tool for the heperHDR. I'ts will arrive to me in few weeks, so then i'll test it and share results.

**Comment by games133** [comment]
hi ive goit a dennon aavr and thinking of using the second out put to connect to a grabber has anyone tried that? i did use it on a lytmi system. whitch allowed me to use earc and cec and hdcp 2.2. im in middle of trying to build this set up.

**Comment by JustMLC4real** [comment]
Can you confirm that it can capture HDCP content? I want to record my PS3 and I wonder if the capture card you use supports HDCP recording
